pkgs/games/build-support/build-game.nix

```diff
@@ -30,7 +29,7 @@ buildSandbox (stdenv.mkDerivation ({
   # Try to evade tarbombs
   setSourceRoot = ''
     popd &> /dev/null
-  '' + lib.optionalString (setSourceRoot == "") ''
+  '' + attrs.setSourceRoot or ''
```

Comment:
What's the reason for this change to be included in the main commit adding the game?

Reply:
None. It was part of another change and somehow landed here.

Reply:
This is still not resolved, could you please remove this change from the commit?

Reply:
Oh, crap ;-)

Reply:
Done !